riva/proto/riva_asr.proto

```diff
@@ -111,6 +111,17 @@ message StreamingRecognizeRequest {
   RequestId id = 100;
 }
 
+/*
+ * EndpointingConfig is used for configure all end of utterance related values
```

Comment:
Update comment to:

EndpointingConfig is used for configuring different fields related to start or end of utterance

Reply:
Updated in latest commit. Please check